### ISSUE TITLE
WP SMUSH

### ISSUE THREAD

**Issue by DirkJahn** [AyeCode/geodirectory#330]
I shortly tested WP smush, as you wrote in some posts that pic optimization is integrated into GDv2. With WP Smush I see some serious issues:

1) Only featured images are found to smush.
2) On upload smushing not working (even for featued images)

**Comment by Stiofan** [comment]
This will be worked on once bugs are cleared.

**Comment by picocodes** [comment]
@Stiofan did the team behind wp smush respond to the email?

**Comment by Stiofan** [comment]
Nope, no response.  I have tried tweeting them, if not luck we might have to move on to the next biggest competitor.

**Comment by Stiofan** [comment]
ok they tweeted back, https://twitter.com/wpmudev/status/1126790118826749952
Lets put a diff file together for the changes we need?

**Comment by 1naveengiri** [comment]
just a reminder.  seems a lost ticket.

**Comment by Stiofan** [comment]
@kprajapatii lets get this done, even if we can't hook into the main process maybe we can use their functions and build our own UI?
